templates/managed-ec2-win-v5.j2

```diff
@@ -0,0 +1,335 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  Provision EC2 instance, connect to Jumpcloud, and associate with a jumpcloud systems group.
+Parameters:
+  MaintenanceWindow:
```

Comment:
ditto here.

Reply:
OK, agreed- I updated this PR with a commit that removes this MaintenanceWindow param